doc/nrf/app_dev/device_guides/thingy91x/thingy91x_recover_to_factory_firmware.rst

```diff
@@ -0,0 +1,75 @@
+.. _thingy91x_recover_to_factory_firmware:
+
+Recovering the Thingy:91 X to factory firmware
+##############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can recover the Thingy:91 X to factory firmware using the ``nrfutil device`` command in the `nRF Util commands <Installing and upgrading nRF Util commands_>`_.
+
+If your Thingy:91 X is powered on and connected using a USB cable, it shows up as a USB device.
+Else, something is preventing the :ref:`connectivity_bridge` application on the nRF5340 from running.
+For example, this could be an application running on the nRF9151 that is driving the wrong pins.
+Another typical case is that the nRF5340 has been flashed by accident.
+
+You need to use an external J-Link to follow these steps.
+You can also use a Nordic development kit with a debug-out port.
+
+Complete the following steps to recover the Thingy:91 X to factory firmware:
+
+#. Download the following files from the `hello-nrfcloud/firmware release v2.0.1 <hello-nrfcloud v2.0.1_>`_:
+
+   * nRF5340 application firmware - :file:`connectivity-bridge-v2.0.1-thingy91x-nrf53-app.hex`
+   * nRF5340 network firmware - :file:`connectivity-bridge-v2.0.1-thingy91x-nrf53-net.hex`
+   * nRF9151 firmware - :file:`hello.nrfcloud.com-v2.0.1+debug-thingy91x-nrf91.hex`
+
+#. Set the SWD switch to **nRF91** and run the ``nrfutil device recover`` command to recover the nRF9151.
+#. Set the SWD switch to **nRF53** and repeat the same for the nRF5340.
+
+   You can use the ``nrfutil device device-info`` command to check that you are connected to the correct chip.
+   When using a DK, there is an ambiguity between targeting the on-board chip and using debug-out.
+   Running the ``nrfutil device device-info`` command is recommended to make sure you are not programming the on-board target accidentally.
```

Comment:
Can this be a note? Then you don't need to repeat it as separate steps later on.